src/v1/account_validity.rs

```diff
@@ -0,0 +1,59 @@
+//! [GET /_synapse/admin/v1/account_validity/validity](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/account_validity.rst)
+
+use std::time::SystemTime;
+
+use ruma::api::ruma_api;
+use ruma::identifiers::UserId;
+
+ruma_api! {
+    metadata: {
+        description: "Extend the account validity.",
+        method: POST,
+        name: "set_account_validity",
+        path: "/_synapse/admin/v1/account_validity/validity",
+        rate_limited: false,
+        authentication: None, // AccessToken?
```

Comment:
This endpoint seems very unlikely to work without authentication 😉

```suggestion
        authentication: AccessToken,
```